### ISSUE TITLE
Documentation issues with turnjs 4

### ISSUE THREAD

**Issue by manelephant** [blasten/turn.js#287]
Hello,

Don't know if this is the right place for this request.

Where is the turnjs 4 documentation to get what you see on turnjs.com : Book shelves with click handler opening a fullscreen layer with the flipbook + social media and close button ?

There is only basics examples in "samples" folder compared to what made me buy a license.

Thanx

**Comment by blasten** [comment]
Hello,

The shelf wasn't included in the package of turn.js 4th release, but I do think it should be part of.   Are you a web developer?

**Comment by manelephant** [comment]
Hello,

Your lib is great, no problem about that, the only small deception is that i was expecting to get ressources to reproduce what you see on your website :
- Social media button
- Books shelves
- Table of content shortcut

Well in short it would be cool to add a new sample with this code.

I am a developer so i'll reproduce those elements by hand.

Thank you

**Comment by blasten** [comment]
@H-Works I will go over those points.

**Comment by jordanmaslyn** [comment]
Hey @blasten, first off thank you for all of your work on this awesome lib! 

Are there any updates on getting a tutorial set up to accomplish the bookshelf concept on your project's homepage? This was something I kind of anticipated being provided as well based on it being demo-ed on the homepage.

@H-Works If you were able to get this up and running, is it something you'd be willing to provide some guidance on?

**Comment by blasten** [comment]
yeah! I started to write the new docs

**Comment by manelephant** [comment]
Hi @jordanmaslyn i didn't had time to make the shelf animation so i can't really help here.

**Comment by Match0707** [comment]
Hi, I also have just purchased the 4th version for a project at my company, expecting it would have all the things that were demonstrated on the website. When will the new samples and docs be ready? @blasted, is there anything you can share with me in the meantime, since I need to get it into a project in the next few weeks. I tried replicating the bookshelf effect by looking at the source code on your website, but no luck yet. Any simple little code sample will be greatly appreciated. Thanks!

**Comment by RobeeeJay** [comment]
I purchased the 4th version expecting to have an easy way to open a book in the same animated style as the bookshelf demo. :(

The flipbook is great, and works nicely on mobile devices (apart from a zoom panning issue on my iPad which I can't work out), but it would have been great if it included the animated opening that the homepage has. :/

**Comment by shahdanish** [comment]
RobeeJay don't know it is right place for such request but i need 4the version of flip book.. 
will you do a favour??

**Comment by RobeeeJay** [comment]
Certainly, go to http://turnjs.com/get and click "Order now", then pay $99 and you'll get a copy from the author sent to you. :p

**Comment by KnightAlex** [comment]
Hi! Love the library. (We have bought v4 license)

We are also after sample code for the bookshelf. Did you make any progress on the documentation for this? I can't see anything in the API docs, or in the samples provided. We are also after the table of contents shortcut.

Thanks!
